log/logClient.go

```diff
@@ -204,6 +205,39 @@ func (fd *Feeder) WatchAlerts(logPath string, jsonFormat bool) error {
 			break
 		}
 
+		if namespace != "" {
+			re, err := regexp.Compile(namespace)
+			if err != nil {
+				return err
+			}
+			val := re.MatchString(res.NamespaceName)
+			if !val {
+				return nil
+			}
+		}
+
+		if logType != "" {
+			re, err := regexp.Compile(logType)
+			if err != nil {
+				return err
+			}
+			val := re.MatchString(res.Type)
+			if !val {
+				return nil
+			}
+		}
+
+		if operation != "" {
+			re, err := regexp.Compile(operation)
+			if err != nil {
+				return err
+			}
+			val := re.MatchString(res.Operation)
+			if !val {
+				return nil
+			}
```

Comment:
Can we make the regex match case insensitive?

I tried it locally, 
` ./karmor log --operation=Network` works as expected but
` ./karmor log --operation=network` fails.

Reply:
yeah, will implement it